styleguide/haml_locals.rb

```diff
@@ -182,6 +182,62 @@
     HTML
   },
 
+  "footer" => {
```

Comment:
Any way we can do this without the locals? Nice if we can reduce dependance on this file over time

Reply:
There's a couple of ways and the pain on working on this feature is making them seem like something I might explore sooner rather than later. In the end I decided to stick with using this until the Header is finished then dedicate some time to improving the dev experience.